### PR TITLE
distortion_on_tied_notes

### PR DESCRIPTION
**Pull request by RomanPudashkin**
Resolves: https://github.com/musescore/MuseScore/issues/21195
Resolves: https://github.com/musescore/MuseScore/issues/21963

### CODE REVIEW

File: src/framework/mpe/tests/multinotearticulationstest.cpp [hunk 1]
```diff
@@ -30,7 +30,7 @@ using namespace mu;
 using namespace mu::mpe;
 using namespace mu::mpe::tests;
 
-class Engraving_MultiNoteArticulationsTest : public ::testing::Test
+class MultiNoteArticulationsTest : public ::testing::Test
```
Comment (cbjeukendrup) [inline src/framework/mpe/tests/multinotearticulationstest.cpp:33]: I believe Igor named them that way for nicer sorting in Qt Creator

Comment (RomanPudashkin) [reply src/framework/mpe/tests/multinotearticulationstest.cpp]: Thanks, I didn't realize that was intentional

**Comment by DmitryArefiev** [comment]
Tested #21195 #21963 on Win10 - **FIXED**

There are some minor issues (comparing with 4.4 master), I'll log them separately

**Comment by atirut-w** [comment]
Can this get a backport for 4.2.x?

**Comment by cbjeukendrup** [comment]
@atirut-w This fix is included in 4.3, which will be released early next week. That also means that there won't be any 4.2.x update anymore, so a backport would not really be useful.

**Pull request merged**